glean-core/python/glean/glean.py

```diff
@@ -284,7 +284,9 @@ def _reset(cls) -> None:
         # Importantly on Windows, this closes the handle to the database so
         # that the data directory can be deleted without a multiple access
         # violation.
-        _uniffi.glean_test_destroy_glean(False)
+        # With `clear_stores=False` the path will never be touched,
+        # so we can make up one.
+        _uniffi.glean_test_destroy_glean(False, "./:/!/path/that/can/never/exist")
```

Comment:
fwiw I currently _dislike_ the different behavior between the different languages and before I land this I will probably rewrite this part.